service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/CreateAiNotebookInstanceStepTest.java

```diff
@@ -31,6 +54,79 @@ public class CreateAiNotebookInstanceStepTest extends BaseUnitTest {
   private static final String SERVER_ID = "test-server";
   private static final String NOTEBOOK_DISABLE_ROOT_KEY = "notebook-disable-root";
 
+  //  @Mock private
+  @Mock private FlightContext mockFlightContext;
+  @Mock private CrlService mockCrlService;
+  @Mock private GcpCloudContextService mockGcpCloudContextService;
+
+  @Mock private AIPlatformNotebooksCow mockAIPlatformNotebookCow;
+  @Mock private AIPlatformNotebooksCow.Instances mockInstances;
+  @Mock private AIPlatformNotebooksCow.Instances.Create mockInstancesCreate;
+
+  @Mock private AIPlatformNotebooksCow.Operations mockOperations;
+  @Mock private OperationCow<Operation> mockOperationCow;
+  @Mock private OperationCow.OperationAdapter<Operation> mockOperationAdapter;
+  @Mock private OperationCow.OperationAdapter.StatusAdapter mockOperationStatusAdapter;
+  @Mock private Operation mockOperation;
+
```

Comment:
@yuhuyoyo Should I split the mock notebook COW stuff into a separate test file or leave it here?

I would prefer having a `setup()` function, but the other tests don't use the COW.